scripts.d/50-vvenc.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+SCRIPT_REPO="https://github.com/fraunhoferhhi/vvenc"
```

Comment:
I normally include the .git in all those URLs.

Reply:
Added ".git" after script repo url.